src/main/java/com/solvd/elasticsearchmicroservice/kafka/handler/PutHandler.java

```diff
@@ -0,0 +1,9 @@
+package com.solvd.elasticsearchmicroservice.kafka.handler;
+
+import com.solvd.elasticsearchmicroservice.kafka.event.NoteEvent;
+
+public interface PutHandler extends Handler {
```

Comment:
I suggest you the following design for your classes:

```java
interface Handler {
void handle();
}

class Put implements Handler {}

class Post implements Handler {}

class Delete implements Handler {}
```

WDYT?

Reply:
https://github.com/vpetrovaa/elasticsearch-microservice/pull/4/commits/6e181c7f8c2d56295c557e86d5cc8b6f3a87ed84